packages/apps/src/SideBar/ChainInfo.tsx

```diff
@@ -24,35 +24,61 @@ function ChainInfo ({ className = '', isToggled, onClick }: Props): React.ReactE
   const runtimeVersion = useCall<RuntimeVersion>(api.rpc.state.subscribeRuntimeVersion, []);
 
   return (
-    <div
-      className={`apps--SideBar-logo${onClick ? ' isClickable' : ''} ${className} ui--highlight--border`}
-      onClick={onClick}
-    >
-      <div className='apps--SideBar-logo-inner'>
-        <ChainImg />
-        <div className='info'>
-          <Chain className='chain' />
-          {runtimeVersion && (
-            <div className='runtimeVersion'>{t<string>('version {{version}}', { replace: { version: runtimeVersion.specVersion.toNumber() } })}</div>
+    <div className={className}>
+      <div className='apps--Sidebar-top'>{'polkadot{.js}'}</div>
```

Comment:
I would remove this label completely, see for instance https://portal.chain.centrifuge.io/#/explorer